From 2f91ce74e9c323349b95cce95ad5f8ef2e1e97d4 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Thu, 29 Apr 2021 18:39:23 +0000
Subject: [PATCH] disable ninjalog upload

---
 autoninja | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/autoninja b/autoninja
index b9990ee8a..12cbe2731 100755
--- a/autoninja
+++ b/autoninja
@@ -30,12 +30,12 @@ if eval "$command"; then
   fi
 
   # Collect ninjalog from googler.
-  python3 "$(dirname -- "$0")/ninjalog_uploader_wrapper.py" --cmd $command
+  # python3 "$(dirname -- "$0")/ninjalog_uploader_wrapper.py" --cmd $command
   exit
 fi
 
 # Collect ninjalog from googler.
-python3 "$(dirname -- "$0")/ninjalog_uploader_wrapper.py" --cmd $command
+# python3 "$(dirname -- "$0")/ninjalog_uploader_wrapper.py" --cmd $command
 
 # Return an error code of 1 so that if a developer types:
 # "autoninja chrome && chrome" then chrome won't run if the build fails.
-- 
2.30.2

