From abc1619e40dc49f4f5feec997636837a0a0d8606 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@linux.com>
Date: Wed, 14 Apr 2021 16:56:29 +0000
Subject: [PATCH 3/3] weston include dir

---
 cmake/package.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/package.cmake b/cmake/package.cmake
index 96ad106..ce198b8 100644
--- a/cmake/package.cmake
+++ b/cmake/package.cmake
@@ -32,6 +32,7 @@ else()
     string(REPLACE "." ";" WESTON_VERSION_LIST ${WESTON_VERSION})
     list(GET WESTON_VERSION_LIST 0 WESTON_VERSION_MAJOR)
     pkg_check_modules(LIBWESTON REQUIRED libweston-${WESTON_VERSION_MAJOR})
+    include_directories(${LIBWESTON_INCLUDE_DIRS})
   endif()
 endif()
 
-- 
2.30.2

