From a979ff5bf4e4cd729ae30b586b6f870d5d6190a4 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@toyotaconnected.com>
Date: Wed, 31 Jul 2024 18:05:18 -0700
Subject: [PATCH] ivi homescreen support

Signed-off-by: Joel Winarske <joel.winarske@toyotaconnected.com>
---
 lib/logic/app_logic.dart | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/lib/logic/app_logic.dart b/lib/logic/app_logic.dart
index 100b0b5..9ee18fb 100644
--- a/lib/logic/app_logic.dart
+++ b/lib/logic/app_logic.dart
@@ -93,15 +93,11 @@ class AppLogic {
 
   /// Called from the UI layer once a MediaQuery has been obtained
   void handleAppSizeChanged(Size appSize) {
-    /// Disable landscape layout on smaller form factors
-    bool isSmall = display.size.shortestSide / display.devicePixelRatio < 600;
-    supportedOrientations = isSmall ? [Axis.vertical] : [Axis.vertical, Axis.horizontal];
+    supportedOrientations = [Axis.vertical, Axis.horizontal];
     _updateSystemOrientation();
     _appSize = appSize;
   }
 
-  Display get display => PlatformDispatcher.instance.displays.first;
-
   bool shouldUseNavRail() => _appSize.width > _appSize.height && _appSize.height > 250;
 
   void _updateSystemOrientation() {
-- 
2.25.1

