Index: src/taffy_ffi/Cargo.toml
===================================================================
--- src.orig/taffy_ffi/Cargo.toml
+++ src/taffy_ffi/Cargo.toml
@@ -7,8 +7,7 @@ edition = "2021"
 crate-type = ["staticlib"]
 
 [dependencies]
-# taffy = { version = "0.3.13", default-features = false, features = ["grid"] }
-taffy = { git = "https://github.com/DioxusLabs/taffy", rev = "daa07e0", default-features = false, features = ["flexbox", "grid", "taffy_tree"] }
+taffy = { path = "../../taffy", default-features = false, features = ["flexbox", "grid", "taffy_tree"] }
 
 [profile.minimize]
 inherits = "release"
@@ -17,4 +16,4 @@ panic = "abort"
 strip = true
 lto = "fat"
 
-[workspace]
\ No newline at end of file
+[workspace]
