From 5b050dc0b7fb21fd5752ca98b434950791400df2 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Wed, 12 Jun 2024 06:40:59 -0700
Subject: [PATCH 1/5] error ignoring return value of function declared wit

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 libs/utils/include/utils/memalign.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/utils/include/utils/memalign.h b/libs/utils/include/utils/memalign.h
index 4c048bffd..a96a08a70 100644
--- a/libs/utils/include/utils/memalign.h
+++ b/libs/utils/include/utils/memalign.h
@@ -40,7 +40,7 @@ inline void* aligned_alloc(size_t size, size_t align) noexcept {
 #if defined(WIN32)
     p = ::_aligned_malloc(size, align);
 #else
-    ::posix_memalign(&p, align, size);
+    (void)::posix_memalign(&p, align, size);
 #endif
     return p;
 }
-- 
2.45.1

