From 204603cd9ec35885e120c8b5dd8fb8e87caf624e Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarsk@gmail.com>
Date: Mon, 20 Mar 2023 19:13:12 -0700
Subject: [PATCH] remove angle

Signed-off-by: Joel Winarske <joel.winarsk@gmail.com>
---
 tools/gn | 29 -----------------------------
 1 file changed, 29 deletions(-)

diff --git a/flutter/tools/gn b/flutter/tools/gn
index 27f6598..40b2a2b 100755
--- a/flutter/tools/gn
+++ b/flutter/tools/gn
@@ -554,35 +554,6 @@ def to_gn_args(args):
   if args.prebuilt_impellerc is not None:
     gn_args['impeller_use_prebuilt_impellerc'] = args.prebuilt_impellerc
 
-  # ANGLE is exclusively used for:
-  #  - Windows at runtime
-  #  - Non-fuchsia host unit tests (is_host_build evaluates to false).
-  # Setting these variables creates warnings otherwise.
-  # If we add ANGLE usage on other platforms, include them here.
-  # There is a special case for Android on Windows because there we _only_ build
-  # gen_snapshot, but the build defines otherwise make it look like the build is
-  # for a host Windows build and make GN think we will be building ANGLE.
-  if is_host_build(args) or (args.target_os == 'android' and
-                             get_host_os() == 'win'):
-    # Do not build unnecessary parts of the ANGLE tree.
-    gn_args['angle_build_all'] = False
-    gn_args['angle_has_astc_encoder'] = False
-    # Force ANGLE context checks on Windows to prevent crashes.
-    # TODO(loic-sharma): Remove this once ANGLE crashes have been fixed.
-    # https://github.com/flutter/flutter/issues/114107
-    if get_host_os() == 'win':
-      gn_args['angle_force_context_check_every_call'] = True
-
-    # Requires RTTI. We may want to build this in debug modes, punting on that
-    # for now.
-    gn_args['angle_enable_vulkan_validation_layers'] = False
-    gn_args['angle_vulkan_headers_dir'
-           ] = '//third_party/vulkan-deps/vulkan-headers/src'
-    gn_args['angle_vulkan_loader_dir'
-           ] = '//third_party/vulkan-deps/vulkan-loader/src'
-    gn_args['angle_vulkan_tools_dir'
-           ] = '//third_party/vulkan-deps/vulkan-tools/src'
-
   return gn_args
 
 
-- 
2.39.2

