From 3a7d643ff33d791cbee9423d044060b89ca7a3de Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarsk@gmail.com>
Date: Sun, 5 Mar 2023 10:37:44 -0800
Subject: [PATCH] Disable pre-canned sysroot

Signed-off-by: Joel Winarske <joel.winarsk@gmail.com>
---
 build/config/sysroot.gni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/config/sysroot.gni b/build/config/sysroot.gni
index 7987e51..1de6942 100644
--- a/build/config/sysroot.gni
+++ b/build/config/sysroot.gni
@@ -14,7 +14,7 @@ declare_args() {
 
   # Whether to use the default sysroot when building for Linux, if an explicit
   # sysroot isn't set.
-  use_default_linux_sysroot = true
+  use_default_linux_sysroot = false
 }
 
 if (current_toolchain == default_toolchain && target_sysroot != "") {
-- 
2.39.2

