From bfe7fc3564130ecda2e62564ec65a47d76ac0de2 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Thu, 4 Jul 2024 06:08:09 -0700
Subject: [PATCH] disable x11

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 build_overrides/vulkan_headers.gni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flutter/build_overrides/vulkan_headers.gni b/flutter/build_overrides/vulkan_headers.gni
index 5efa852344..8a4144db20 100644
--- a/flutter/build_overrides/vulkan_headers.gni
+++ b/flutter/build_overrides/vulkan_headers.gni
@@ -6,6 +6,6 @@
 # set anything.
 
 if (is_linux) {
-  vulkan_use_x11 = true
+  vulkan_use_x11 = false
   vulkan_use_wayland = true
 }
-- 
2.43.5

