From 180684d091e578d33753f89cb911a8c19b6a6776 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Sat, 2 Apr 2022 11:50:33 -0700
Subject: [PATCH] allow deprecated calls

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 tools/gn | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/flutter/tools/gn b/flutter/tools/gn
index 7623c3b4ee..cca1f8a5f5 100755
--- a/flutter/tools/gn
+++ b/flutter/tools/gn
@@ -128,6 +128,9 @@ def to_gn_args(args):
 
     gn_args = {}
 
+    # https://github.com/flutter/flutter/issues/93871
+    gn_args['allow_deprecated_api_calls'] = True
+
     if args.bitcode:
       if args.target_os != 'ios':
         raise Exception('Bitcode is only supported for iOS')
--
2.34.1
