From 6676431f3324357b8daead2ae00c4b88f24f6449 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Thu, 4 Jul 2024 06:06:19 -0700
Subject: [PATCH] gn riscv32 and riscv64

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 tools/gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flutter/tools/gn b/flutter/tools/gn
index 0be76e5477..b28a17ec15 100755
--- a/flutter/tools/gn
+++ b/flutter/tools/gn
@@ -119,7 +119,7 @@ def can_use_prebuilt_dart(args):
       prebuilt = 'macos-x64'
     else:
       prebuilt = 'linux-x64'
-  elif args.target_os == 'linux' and args.linux_cpu in ['x64', 'arm64']:
+  elif args.target_os == 'linux' and args.linux_cpu in ['x64', 'arm64', 'riscv32', 'riscv64']:
     prebuilt = 'linux-%s' % args.linux_cpu
   elif args.target_os == 'mac' and args.mac_cpu in ['x64', 'arm64']:
     prebuilt = 'macos-%s' % args.mac_cpu
-- 
2.43.5

