diff --git a/build/config/sysroot.gni b/build/config/sysroot.gni
index fb0f3c3..69bbf7e 100644
--- a/build/config/sysroot.gni
+++ b/build/config/sysroot.gni
@@ -27,7 +27,7 @@ if (current_toolchain == default_toolchain && target_sysroot != "") {
 } else if (is_linux && !is_chromeos) {
   if (current_cpu == "mipsel") {
     sysroot = rebase_path("//mipsel-sysroot/sysroot")
-  } else if (use_default_linux_sysroot && !is_fuchsia && current_cpu == "x64") {
+  } else if (use_default_linux_sysroot && !is_fuchsia && target_cpu == "x64") {
     sysroot = rebase_path("//build/linux/debian_sid_amd64-sysroot")
     assert(
         exec_script("//build/dir_exists.py", [ sysroot ], "string") == "True",
