From a34e06e871322a427741a88e67995d67847761ef Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Wed, 23 Feb 2022 13:42:13 -0800
Subject: [PATCH] path updates

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 src/flutter-pi.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/flutter-pi.c b/src/flutter-pi.c
index 46ab470..4149626 100644
--- a/src/flutter-pi.c
+++ b/src/flutter-pi.c
@@ -2260,7 +2260,7 @@ static bool setup_paths(void) {
     }
     
     asprintf(&kernel_blob_path, "%s/kernel_blob.bin", flutterpi.flutter.asset_bundle_path);
-    asprintf(&app_elf_path, "%s/app.so", flutterpi.flutter.asset_bundle_path);
+    asprintf(&app_elf_path, "%s/libapp.so", flutterpi.flutter.asset_bundle_path);
 
     if (flutterpi.flutter.runtime_mode == kDebug) {
         if (!PATH_EXISTS(kernel_blob_path)) {
@@ -2269,14 +2269,14 @@ static bool setup_paths(void) {
         }
     } else if (flutterpi.flutter.runtime_mode == kRelease) {
         if (!PATH_EXISTS(app_elf_path)) {
-            LOG_ERROR("Could not find \"app.so\" file inside \"%s\", which is required for release and profile mode.\n", flutterpi.flutter.asset_bundle_path);
+            LOG_ERROR("Could not find \"libapp.so\" file inside \"%s\", which is required for release and profile mode.\n", flutterpi.flutter.asset_bundle_path);
             return false;
         }
     }
 
-    asprintf(&icu_data_path, "/usr/lib/icudtl.dat");
+    asprintf(&icu_data_path, "/usr/share/flutter/icudtl.dat");
     if (!PATH_EXISTS(icu_data_path)) {
-        LOG_ERROR("Could not find \"icudtl.dat\" file inside \"/usr/lib/\".\n");
+        LOG_ERROR("Could not find \"icudtl.dat\" file inside \"/usr/share/flutter/\".\n");
         return false;
     }
 
-- 
2.31.1
