From cf9bd4cf95decc2d81282c9f5784bc47c22802bf Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@toyotaconnected.com>
Date: Thu, 25 Feb 2021 02:31:43 +0000
Subject: [PATCH] autogen problem

---
 brcm/CMakeLists.txt | 25 +++++++++++++------------
 1 file changed, 13 insertions(+), 12 deletions(-)

diff --git a/brcm/CMakeLists.txt b/brcm/CMakeLists.txt
index 944843b..8aaffcd 100644
--- a/brcm/CMakeLists.txt
+++ b/brcm/CMakeLists.txt
@@ -11,18 +11,19 @@ file(GLOB brcmSrc
 
 find_program(python REQUIRED)
 
-execute_process(COMMAND "python" "${CMAKE_CURRENT_SOURCE_DIR}/cle/gen_pack_header.py" "${CMAKE_CURRENT_SOURCE_DIR}/cle/v3d_packet_v21.xml" "21" OUTPUT_VARIABLE V3D_PACKET_V21_PACK ERROR_VARIABLE CMD_ERROR)
-if(CMD_ERROR STREQUAL "")
-	file(WRITE "${CMAKE_CURRENT_SOURCE_DIR}/cle/v3d_packet_v21_pack.h" "${V3D_PACKET_V21_PACK}" )
-else(CMD_ERROR STREQUAL "")
-	message(FATAL_ERROR ${CMD_ERROR})
-endif(CMD_ERROR STREQUAL "")
-execute_process(COMMAND "python" "${CMAKE_CURRENT_SOURCE_DIR}/cle/gen_zipped_file.py" "${CMAKE_CURRENT_SOURCE_DIR}/cle/v3d_packet_v21.xml" OUTPUT_VARIABLE V3D_XML ERROR_VARIABLE CMD_ERROR2)
-if(CMD_ERROR2 STREQUAL "")
-	file(WRITE "${CMAKE_CURRENT_SOURCE_DIR}/cle/v3d_xml.h" "${V3D_XML}" )
-else(CMD_ERROR2 STREQUAL "")
-	message(FATAL_ERROR ${CMD_ERROR2})
-endif(CMD_ERROR2 STREQUAL "")
+add_custom_command(
+  OUTPUT cle/v3d_packet_v21_pack.h
+  COMMAND python cle/gen_pack_header.py cle/v3d_packet_v21.xml 21 > cle/v3d_packet_v21_pack.h
+  DEPENDS cle/gen_pack_header.py cle/v3d_packet_v21.xml
+  WORKING_DIR ${CMAKE_CURRENT_SOURCE_DIR}
+)
+
+add_custom_command(
+  OUTPUT cle/v3d_xml.h
+  COMMAND python /cle/gen_zipped_file.py cle/v3d_packet_v21.xml > cle/v3d_xml.h
+  DEPENDS /cle/gen_zipped_file.py cle/v3d_packet_v21.xml
+  WORKING_DIR ${CMAKE_CURRENT_SOURCE_DIR}
+)
 
 add_library(brcm OBJECT ${brcmSrc})
 target_compile_definitions(brcm PRIVATE V3D_VERSION=21)
-- 
2.29.2

