From a02e26fc91df3af043755502b628fcabc4c9edd2 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@linux.com>
Date: Wed, 24 Feb 2021 19:57:26 +0000
Subject: [PATCH] disable glfw default features

---
 flutter-glfw/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flutter-glfw/Cargo.toml b/flutter-glfw/Cargo.toml
index 55bf779..f72cd0b 100644
--- a/flutter-glfw/Cargo.toml
+++ b/flutter-glfw/Cargo.toml
@@ -15,4 +15,4 @@ flutter-plugins = { path = "../flutter-plugins" }
 locale_config = "0.3.0"
 log = "0.4.8"
 parking_lot = "0.10.0"
-glfw = "0.37.0"
\ No newline at end of file
+glfw = { version = "0.37.0", default-features = false }
-- 
2.29.2
