From a34415371e6e78e3acf3a07cf6b9c6c5e6d7dbc8 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@linux.com>
Date: Wed, 24 Feb 2021 20:02:01 +0000
Subject: [PATCH] gl_proc resolver type

---
 flutter-glfw/src/handler.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flutter-glfw/src/handler.rs b/flutter-glfw/src/handler.rs
index 81e01b7..8ce8af9 100644
--- a/flutter-glfw/src/handler.rs
+++ b/flutter-glfw/src/handler.rs
@@ -64,7 +64,7 @@ impl FlutterOpenGLHandler for GlfwOpenGLHandler {
         true
     }
 
-    fn gl_proc_resolver(&self, proc: *const i8) -> *mut c_void {
+    fn gl_proc_resolver(&self, proc: *const u8) -> *mut c_void {
         unsafe { glfw::ffi::glfwGetProcAddress(proc as _) as _ }
     }
 }
-- 
2.29.2

